### PR TITLE
Add ability for the nginx reload command to be configurable

### PR DESCRIPTION
**Pull request by samdjstevens**
My final obstacle with using this plugin with a Homebrew install of NGINX on macOS is that the a different command is needed to reload NGINX.

This PR brings in another optional config value for the command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,12 @@ If you want to change the location of the managed nginx configuration file, set
 
     config.reverse_proxy.nginx_config_file = '/usr/local/etc/nginx/vagrant-proxy-config'
 
+### Specifying the NGINX reload command
+
+After the NGINX configuration file is generated, a reload command is executed so that the changes take affect. By default the command executed is `sudo service nginx reload`. If you need to change this, set the `config.reverse_proxy.nginx_reload_command` option to the command to be executed:
```
Comment (sjamaan) [inline README.md]: The noun "affect" has a meaning related to psychology. I think you mean "effect" :)

File: lib/vagrant-reverse-proxy/action/write_nginx_config.rb [hunk 2]
```diff
@@ -66,9 +66,12 @@ def call(env)
             end
           end
 
-          # Finally, copy tmp config to actual config and reload nginx
+          # Finally, copy tmp config to actual config
           Kernel.system('sudo', 'cp', tmp_file.to_s, nginx_config_file)
-          Kernel.system('sudo', 'service', 'nginx', 'reload')
+
+          # And reload nginx
+          nginx_reload_command = env[:machine].config.reverse_proxy.nginx_reload_command || 'sudo service nginx reload'
```
Comment (sjamaan) [inline lib/vagrant-reverse-proxy/action/write_nginx_config.rb]: Actually, I think your example from the docs (`sudo nginx -s reload`) is a better default. It should work on Linux as well as on Mac OS and the BSDs, which means one would need to override it much less often.

**Review by sjamaan — CHANGES_REQUESTED** [review]
Looks good, except for two small nitpicks

**Comment by samdjstevens** [comment]
No worries - changed the default command/example and fixed the grammar issue in the README.

**Review by sjamaan — APPROVED** [review]

**Comment by sjamaan** [comment]
Thanks a lot!

**Comment by sjamaan** [comment]
You can now install this, it's been published as gem version 0.3.1

**Comment by samdjstevens** [comment]
Nice one, thanks!

**Pull request merged**